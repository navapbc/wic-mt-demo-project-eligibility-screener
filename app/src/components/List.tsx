import { Trans } from 'next-i18next'

import { i18nKey } from '@src/types'

export type ListProps = {
  i18nKeys: i18nKey[]
}

export const List = (props: ListProps) => {
  const { i18nKeys } = props

  return (
    <ul>
      {i18nKeys.map((key, index) => (
        <li key={index}>
          <Trans i18nKey={key} />
        </li>
      ))}
    </ul>
  )
}

export default List
