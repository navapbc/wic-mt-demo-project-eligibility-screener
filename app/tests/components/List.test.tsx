import List from '@components/List'

import { testSnapshot } from '../helpers/sharedTests'

const testProps = {
  i18nKeys: ['list item a', 'list item b', 'list item c'],
}

it('should match snapshot', () => {
  testSnapshot(<List {...testProps} />)
})
