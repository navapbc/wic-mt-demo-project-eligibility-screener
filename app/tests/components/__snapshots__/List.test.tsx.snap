// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<ul>
  <li>
    list item a
  </li>
  <li>
    list item b
  </li>
  <li>
    list item c
  </li>
</ul>
`;
